core-bundle/src/Resources/contao/classes/DataContainer.php

```diff
@@ -776,8 +776,8 @@ protected function generateButtons($arrRow, $strTable, $arrRootIds=array(), $bln
 			{
 				if (\is_array($v['label']))
 				{
-					$label = $v['label'][0];
-					$title = sprintf($v['label'][1], $id);
+					$label = $v['label'][0] ?? $k;
+					$title = sprintf($v['label'][1] ?? $k, $id);
```

Comment:
I think this could cause problems, because `$k` does not necessarily contain a specifier (sprintf).

Reply:
I don't think it can cause problems, but it is at the least a behaviour change. Should be changed to:

```suggestion
					$title = sprintf($v['label'][1] ?? '', $id);
```

Reply:
That's right, I remembered that `sprintf` can't handle strings that don't contain a specifier and return `false`. However, empty values can be handled here, so I prefer the variant to return an empty string to avoid errors. 

[With PHP 8.0 `false` is no longer returned on errors.](https://www.php.net/manual/en/function.sprintf.php#refsect1-function.sprintf-changelog)

Reply:
Hm, I cannot confirm that. It shows no error and does not return `false` in any version in such a case: https://3v4l.org/1DRKh

Reply:
Yes that's right, that was only in my mind 🙃

That's why I prefer your suggestion to return an empty string, since the key (`$k`) may contain values we can't consider.

Reply:
I just used `$k` because it is the initial fallback: https://github.com/contao/contao/blob/d56bc8067cf5b2c3255fd1eed2bdad3766bcf05c/core-bundle/src/Resources/contao/classes/DataContainer.php#L773

It could be solved outside `sprintf` but as @fritzmg confirmed it doesn't matter if there is no specifier included in `$k`